v21.2/query-behavior-troubleshooting.md

```diff
@@ -98,29 +98,29 @@ Throughput is affected by the disk I/O, CPU usage, and network latency. Use the
 
 ## Single hot node
 
-A hot node is one that has much higher resource usage than other nodes. To determine if you have a hot node in your cluster, [access the DB Console](ui-overview.html#db-console-access), click **Metrics** on the left, and navigate to the following graphs. Hover over each of the following graphs to see the per-node values of the metrics. If one of the nodes has a higher value, you have a hot node in your cluster.
+A hot node is one that has much higher resource usage than other nodes. To determine if you have a hot node in your cluster, [access the DB Console](ui-overview.html#db-console-access) and check the following:
 
--   Replication dashboard > Average queries per store graph.
-
--   Overview Dashboard > Service Latency graph
-
--   Hardware Dashboard > CPU percent graph
-
--   SQL Dashboard > SQL Connections graph
-
--   Hardware Dashboard > Disk IOPS in Progress graph
+- Click **Metrics** and navigate to the following graphs. Hover over each of the following graphs to see the per-node values of the metrics. If one of the nodes has a higher value, you have a hot node in your cluster.
```

Comment:
minor nit: Maybe to avoid repeating "following graphs" in the second sentence here, "Hover over each of the graphs to see..." It seems like the first sentence determines which graphs we're referring to?